pkg/apiserver/authenticator/authn.go

```diff
@@ -96,12 +99,20 @@ func New(config AuthenticatorConfig) (authenticator.Request, error) {
 		authenticators = append(authenticators, serviceAccountAuth)
 	}
 
-	if len(config.KeystoneURL) > 0 {
-		keystoneAuth, err := newAuthenticatorFromKeystoneURL(config.KeystoneURL)
-		if err != nil {
-			return nil, err
+	if len(config.KeystoneConfig) > 0 {
```

Comment:
Yeah. but it wasn't doing that. At the time, I think I had both the token plugin and the basicauth plugin combined into one plugin with both types of validators in it. Maybe that was the problem?